src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritConnection.java

```diff
@@ -445,7 +448,9 @@ public void run() {
                     if (!channel.isConnected() || !sshConnection.isConnected()) {
                         throw new IllegalStateException("SSH connection is already lost.");
                     }
-                    sleep(SSH_RX_SLEEP_MILLIS);
+                    if (linecount > 0) {
```

Comment:
could `yield()` be used as an alternate to not starve the CPU? It is generally not recommended to be used, but just lifting the idea.